### DISCUSSION TITLE
Setting app homepage

### DISCUSSION THREAD

**Discussion by PaulMcF1987** [Budibase/budibase#6185] [Help] [answered]
I have looked through the documentation and also feel as though I have searched all areas of the app backend,

Is there a way to set the homepage of an app?

Everytime I open the app, it goes to `app.com/app/table/:id` and displays an empty screen and I then navigate from there.

Is there a way to change this so that it starts in a different screen?

**Comment by SantoshSrinivas79** [comment, marked as answer]
Add a screen with a route to the index page `/`
![screenshot-rocks](https://user-images.githubusercontent.com/1036163/172034385-431dbde1-ddb6-4c12-9bdb-37d91a98adb6.png)
